src/stories/AppMenu.stories.mdx

```diff
@@ -732,7 +732,6 @@ Menu items can be disabled.
         template: '<div class="absolute flex inset-0 p-16"><story /></div>'
       })
     ]}>
-    >
```

Comment:
dormant typo which threw error due to the [mdx2 update](https://storybook.js.org/docs/vue/migration-guide#upgrade-mdx1-to-mdx2)